jam/generator.py

```diff
@@ -138,7 +138,14 @@ def get_field_info(self, field, field_name):
 
 
 class DRFGenerator(Generator):
-    def find_api_and_models(self, api_prefix=None, router_module=None):
+    def model_in_included_apps(self, included_apps, model):
+        if included_apps:
+            matched_apps = filter(lambda x: model.__module__[0:len(x)] == x, included_apps)
```

Comment:
this looks a little low level. See if you can access via model._meta.something... (app_label?)
not fussed if not changed though..